flow/src/main/java/com/sequenceiq/flow/service/FlowService.java

```diff
@@ -49,6 +55,8 @@ public class FlowService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(FlowService.class);
 
+    private static final int FLOW_PAGE_LIMIT = 50;
```

Comment:
this magical 50 is set in 2 places in the code. If these should be the same, we should have one constant for it, or better, have it configured in appl yaml

Reply:
because i changed the logic to use Pageable, this is no longer required from my side. i will remove it